tests/test_runner/test_checkpoint.py

```diff
@@ -146,8 +146,13 @@ def test_get_state_dict():
                         wrapped_model.module.conv.module.bias)
 
 
-def test_load_pavimodel_dist():
+def test_get_torchvision_models():
+    # Check if wrong model paths are given
+    # More details at https://github.com/open-mmlab/mmcv/pull/1668
+    get_torchvision_models()
```

Comment:
This test doesn't seem to be able to test the bug mentioned in PR.

Reply:
If `pkgutil.walk_packages` returns wrong paths, `import_module` returns error because there will not be any real module in the wrong paths.
So, if this test function is done without error, I can assume `pkgutil.walk_packages` returns only correct paths.

I would appreciate it if you could let me know if there is anything lacking.

Reply:
I mean maybe we should add some temp files like `detection/any.oy`, which raise error for the old implementation but perform well for the new implementation.